VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.html

```diff
@@ -206,40 +197,40 @@ <h2 class="govuk-heading-m white">Please wait</h2>
     <div *ngIf="canStartJoinConsultation">
       <a href="javascript:void(0)" class="govuk-button" role="button" draggable="false" id="openStartPCButton"
         data-module="govuk-button" (click)="openStartConsultationModal()">
-        Start a private meeting
+        {{'participant-waiting-room.private-consultation.start' | translate}}
       </a>
 
       <a href="javascript:void(0)" class="govuk-button govuk-button-blue govuk-!-margin-left-5" role="button"
         draggable="false" id="openJoinPCButton" data-module="govuk-button" (click)="openJoinConsultationModal()">
-        Join a private meeting
+        {{'participant-waiting-room.private-consultation.join' | translate}}
       </a>
     </div>
     
     <div *ngIf="canStartJoinConsultation">
       <details class="govuk-details" data-module="govuk-details">
         <summary class="govuk-details__summary" (click)="toggleAccordian()" tabindex="0" role="button" [attr.aria-expanded]="privateConsultationAccordianExpanded">
           <span class="govuk-details__summary-text" id="what-are-private-meetings">
-            What are private meetings?
+            {{'participant-waiting-room.private-consultation.what-are-private-meetings' | translate}}
           </span>
         </summary>
         <div class="govuk-details__text" *ngIf="privateConsultationAccordianExpanded">
           <p class="govuk-body govuk-!-margin-bottom-0">
-            You can talk to other participants in private before the hearing begins or during a break, if you wish.
+            {{'participant-waiting-room.private-consultation.talk-to-other-participants' | translate}}
           </p>
           <ul>
-            <li>Start a meeting by choosing who you want to speak to from the list</li>
-            <li>Join a meeting that's already started by choosing from the list</li>
-            <li>If a meeting room is locked you'll need to be invited by someone in the meeting</li>
+            <li{{'participant-waiting-room.private-consultation.guide1' | translate}}>
+            <li{{'participant-waiting-room.private-consultation.guide2' | translate}}>
+            <li{{'participant-waiting-room.private-consultation.guide3' | translate}}>
           </ul>    
           <p class="govuk-body govuk-!-margin-bottom-0">
-            If you wish to have a private conversation during the hearing you'll need to ask the judge's permission.
+            {{'participant-waiting-room.private-consultation.guide4' | translate}}
           </p>
         </div>
       </details>
     </div>
     
     <div class="govuk-!-margin-top-9">
-      <p class="govuk-body"><strong>Video hearings team: 0300 303 0655</strong></p>
+      <p class="govuk-body"><strong>{{'participant-waiting-room.vh-team' | translate}}</strong></p>
```

Comment:
phone number could be made into a variable
`{ 'participant-waiting-room.vh-team
' | translate:{ phone : '0300 303 0655
' } }}`
then in the translation json would need to replace the number with `{{phone}}`

Reply:
Done